packages/amplify-prompts/src/printer.ts

```diff
@@ -30,8 +30,16 @@ export class AmplifyPrinter implements Printer {
     this.writeLine(`${isHeadless ? '' : '⚠️ '}${chalk.yellow(line)}`);
   };
 
-  error = (line: string): void => {
+  // disable-eslint-next-line @typescript-eslint/no-explicit-any
+  error = (line: string, error?: any): void => {
     this.writeLine(`${isHeadless ? '' : '🛑 '}${chalk.red(line)}`);
+    if (error) {
```

Comment:
to avoid nested if's can we do something like:
```
const errorMessage = error?.message ?? error;
if (errorMessage) {
  this.writeLine(`${chalk.red(errorMessage)}`);
}
```